jdaviz/data/linelists/SDSS-IV.csv

```diff
@@ -1,9 +1,4 @@
 Line Name,Rest Value,Type
-Ly-α,1215.67,Emission
-N V 1240,1240.81,Emission
-C IV 1549,1549.48,Emission
-He II 1640,1640.42,Emission
-C III] 1908,1908.734,Emission

```

Comment:
why are these removed?  Are they also listed in the SDSS entry and just removed here since they're under 2000 A?  Do we need to note that in the changelog or no?

Reply:
Cami requested it in https://stsci.slack.com/archives/C232KCAFR/p1740170111713889?thread_ts=1740157809.231209&cid=C232KCAFR

Reply:
Added change log.